<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddElementaryEvents</class>
 <widget class="QDialog" name="AddElementaryEvents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="listName">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="locale">
       <locale language="Ukrainian" country="Ukraine"/>
      </property>
      <property name="text">
       <string>Додані елементарні події</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="elemEventsList"/>
    </item>
    <item>
     <widget class="QPushButton" name="addEvent">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="locale">
       <locale language="Ukrainian" country="Ukraine"/>
      </property>
      <property name="text">
       <string> Завершити редагування</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addElementaryEvent">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Додати елементарну подію</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>321</width>
     <height>118</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="generousProbabilityName">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Сумма ймовірностей</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="generousProbability">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="elementaryEventName">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="text">
         <string>Ім'я елементарної події</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="addElemEventName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="probabilityName">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="text">
         <string>Ймовірність</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="addProbability"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
